### DISCUSSION TITLE
problema na instalção

### DISCUSSION THREAD

**Discussion by andrelfalcao** [RamonSilva20/mapos#1928] [General]
esta dando esse erro na instalação eu uso o uol host

http://www.falcaotecseg.com.br/mapos/install/index.php

**Comment by eclaytonlima** [comment]
Para habilitar esta funcionalidade, use um editor de texto para modificar a
diretiva allow_url_fopen no arquivo php.ini da seguinte forma:

allow_url_fopen = on


Em seg., 3 de out. de 2022 às 13:49, andrelfalcao ***@***.***>
escreveu:

**Reply by andrelfalcao** [reply]
qual a pasta do arquivo? nao localizei

**Reply by andrelfalcao** [reply]
o arquivo  php.ini  nao localizei em pasta nenhuma...

**Comment by eclaytonlima** [comment]
O arquivo php.ini vc não vai encontrar dentro da pasta mapos, mais sim
dentro da pasta do php. No seu caso como usa hospedagem, tem que entrar no
painel de configurações da sua hospedagem e verificar se está disponível
fazer alterações nas configurações do php.

Em qua, 5 de out de 2022 20:35, andrelfalcao ***@***.***>
escreveu:

**Comment by andrelfalcao** [comment]
ja vi que  vou ter que mudar a hospedagem 
 
 esta dando esse erro agora 
 
 https://www.falcaotecseg.com.br/mapos/install/index.php
 
 
 acho que a versao do banco de dados deles nao bate

**Comment by eclaytonlima** [comment]
Antes de mudar de hospedagem, verifique nas configurações do php geralmente
tem opção de selecionar outra versão do php e ativar.

Em qui, 6 de out de 2022 17:55, andrelfalcao ***@***.***>
escreveu:

**Comment by Rodrigo-Paz** [comment]
Isso mesmo, esta relacionado a versão do PHP.

**Comment by jackanilto** [comment]
Conseguil a instalação meu amigo ? voce vai precisar mudar do PHP Version de 5.6.3 para  7.3.0
Isso voce deve conseguir pelo painel de Controle da sua Hospedagem eu nao sei se a sua empresa perimite fazer estas alterações

Qualquer coisa me diga WhatsApp 22988169294
